[2023-10-22T13:35:08.628+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL.Upload_Data manual__2023-10-22T13:35:06.922713+00:00 [queued]>
[2023-10-22T13:35:08.644+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL.Upload_Data manual__2023-10-22T13:35:06.922713+00:00 [queued]>
[2023-10-22T13:35:08.644+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-22T13:35:08.663+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): Upload_Data> on 2023-10-22 13:35:06.922713+00:00
[2023-10-22T13:35:08.671+0000] {standard_task_runner.py:57} INFO - Started process 236 to run task
[2023-10-22T13:35:08.673+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'Upload_Data', 'manual__2023-10-22T13:35:06.922713+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/first_dags.py', '--cfg-path', '/tmp/tmpudsnx_ql']
[2023-10-22T13:35:08.675+0000] {standard_task_runner.py:85} INFO - Job 33: Subtask Upload_Data
[2023-10-22T13:35:08.743+0000] {task_command.py:416} INFO - Running <TaskInstance: ETL.Upload_Data manual__2023-10-22T13:35:06.922713+00:00 [running]> on host 32031900c769
[2023-10-22T13:35:08.855+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='me' AIRFLOW_CTX_DAG_ID='ETL' AIRFLOW_CTX_TASK_ID='Upload_Data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-22T13:35:06.922713+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-22T13:35:06.922713+00:00'
[2023-10-22T13:35:08.856+0000] {sql.py:274} INFO - Executing: 

        CREATE TABLE scrap (
            harga TEXT,
            alamat TEXT,
            fasilitas TEXT,
            Carport TEXT,
            Dapur TEXT,
            Daya_Listrik TEXT,
            Garasi TEXT,
            Hadap TEXT,
            Hook TEXT,
            ID_Iklan TEXT,
            Jumlah_Lantai TEXT,
            Kamar_Mandi TEXT,
            Kamar_Mandi_Pembantu TEXT,
            Kamar_Pembantu TEXT,
            Kamar_Tidur TEXT,
            Kondisi_Perabotan TEXT,
            Kondisi_Properti TEXT,
            Konsep_dan_Gaya Rumah TEXT,
            Lebar_Jalan TEXT,
            Luas_Bangunan TEXT,
            Luas_Tanah TEXT,
            Material_Bangunan TEXT,
            Material_Lantai TEXT,
            Pemandangan TEXT,
            Ruang_Makan TEXT,
            Ruang_Tamu TEXT,
            Sertifikat TEXT,
            Sumber_Air TEXT,
            Tahun_Di_Renovasi TEXT,
            Tahun_Dibangun TEXT,
            Terjangkau_Internet TEXT,
            Tipe_Properti TEXT
        );

        
[2023-10-22T13:35:08.865+0000] {base.py:73} INFO - Using connection ID 'postgres_docker' for task execution.
[2023-10-22T13:35:08.874+0000] {base.py:73} INFO - Using connection ID 'postgres_docker' for task execution.
[2023-10-22T13:35:08.885+0000] {sql.py:418} INFO - Running statement: 

        CREATE TABLE scrap (
            harga TEXT,
            alamat TEXT,
            fasilitas TEXT,
            Carport TEXT,
            Dapur TEXT,
            Daya_Listrik TEXT,
            Garasi TEXT,
            Hadap TEXT,
            Hook TEXT,
            ID_Iklan TEXT,
            Jumlah_Lantai TEXT,
            Kamar_Mandi TEXT,
            Kamar_Mandi_Pembantu TEXT,
            Kamar_Pembantu TEXT,
            Kamar_Tidur TEXT,
            Kondisi_Perabotan TEXT,
            Kondisi_Properti TEXT,
            Konsep_dan_Gaya Rumah TEXT,
            Lebar_Jalan TEXT,
            Luas_Bangunan TEXT,
            Luas_Tanah TEXT,
            Material_Bangunan TEXT,
            Material_Lantai TEXT,
            Pemandangan TEXT,
            Ruang_Makan TEXT,
            Ruang_Tamu TEXT,
            Sertifikat TEXT,
            Sumber_Air TEXT,
            Tahun_Di_Renovasi TEXT,
            Tahun_Dibangun TEXT,
            Terjangkau_Internet TEXT,
            Tipe_Properti TEXT
        );

        , parameters: None
[2023-10-22T13:35:08.887+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 280, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 392, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 423, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near "TEXT"
LINE 21:             Konsep_dan_Gaya Rumah TEXT,
                                           ^

[2023-10-22T13:35:08.895+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=ETL, task_id=Upload_Data, execution_date=20231022T133506, start_date=20231022T133508, end_date=20231022T133508
[2023-10-22T13:35:08.908+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 33 for task Upload_Data (syntax error at or near "TEXT"
LINE 21:             Konsep_dan_Gaya Rumah TEXT,
                                           ^
; 236)
[2023-10-22T13:35:08.926+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-22T13:35:08.941+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
